R/derive_date_vars.R

```diff
@@ -390,18 +913,14 @@ convert_dtc_to_dt <- function(dtc,
 #'
 #' @param dtc The `'--DTC'` date to convert.
 #'
-#'   A character date is expected in a format like `yyyy-mm-ddThh:mm:ss`.
-#'   A partial datetime will issue a warning.
-#'   Note: you can use [impute_dtc()] function to build a complete datetime.
-#'
-#' @inheritParams impute_dtc
-#'
-#' @author Samia Kabi
+#' @inheritParams impute_dtc_dtm
 #'
 #' @details Usually this computation function can not be used with `%>%`.
 #'
 #' @return A datetime object
 #'
+#' @author Samia Kabi, Stefan Bundfuss
+#'
 #' @keywords computation timing
 #'
```

Comment:
Will need to update keywords and families

Reply:
Updated